Fix no images loading causing a 0% success rate

nate-moo opened the pull request.

nate-moo committed:
Fix no images loading causing a 0% success rate

nate-moo commented:
this works locally, but I don't entirely know if it works if its merged because of how content policies work and stuff

keeloverr commented:
> this works locally, but I don't entirely know if it works if its merged because of how content policies work and stuff

I know nothing about coding but really like this site. Is there a way for me to apply this fix if I'm browsing on Android?

nate-moo commented:
I rewrote the site and added another method of loading the images which you can use.

You can find it here: [https://unlimigur.herokuapp.com](https://unlimigur.herokuapp.com)

The servers aren't strong so it will fail a few requests. If it doesn't load anything refresh your page and it'll hopefully work again. I'm working on making a server less version but that has stagnated because of my partial-hatred of JavaScript and web development and the inability to get the client side stuff to work.

keeloverr commented:
> this works locally, but I don't entirely know if it works if its merged because of how content policies work and stuff

I know nothing about coding but really like this site. Is there a way for me to apply this fix if I'm browsing on Android?

> I rewrote the site and added another method of loading the images which you can use.
> 
> You can find it here: https://unlimigur.herokuapp.com
> 
> The servers aren't strong so it will fail a few requests. If it doesn't load anything refresh your page and it'll hopefully work again. I'm working on making a server less version but that has stagnated because of my partial-hatred of JavaScript and web development and the inability to get the client side stuff to work.

Thank you.
